Documentation/teaching/so2/assign0-kernel-api.rst

```diff
@@ -0,0 +1,120 @@
+=========================
+Assignment 0 - Kernel API
+=========================
+
+-  Deadline: :command:`Wednesday, 24 March 2021, 23:00`
+
+Assignment's Objectives
+=======================
+
+*  getting familiar with the qemu setup
+*  inserting / deleting kernel modules
+*  getting familiar with the list API implemented in the kernel
+*  have fun :)
+
+Statement
+==============
+
```

Comment:
`====` should be as long as the word above.

Reply:
Fixed.